fontforge/python.c

```diff
@@ -9203,6 +9205,7 @@ return( NULL );
 
     excludes = SSFromLayer((PyFF_Layer *) obj);
     if ( PyErr_Occurred() != NULL ) {
+        free(excludes);
```

Comment:
Same with this. How/why are we getting non-NULL with an error?

Reply:
Idem